doc/source/rllib/package_ref/utils.rst

```diff
@@ -248,10 +248,15 @@ Policy utilities
 Other utilities
 ~~~~~~~~~~~~~~~
 
-.. currentmodule:: ray.rllib.utils
+.. currentmodule:: ray.rllib
 
 .. autosummary::
    :nosignatures:
    :toctree: doc/
 
-   tensor_dtype.get_np_dtype
+   utils.tensor_dtype.get_np_dtype
```

Comment:
I'm not sure where to document these, or whitelisting them, so leave this for @sven1977 to help with, thankks

The reason we document these are because they are PublicAPIs

Reply:
Makes sense. Let's leave this here as-is. We'll later retract these (old API stack) APIs' PublicAPI status and then also remove these form the docs again, but for now this is fine.